lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -432,9 +432,16 @@ bool pullCustomize(const PullImportPaths& importedPaths, const UsdMayaPrimUpdate
         const auto&       pulledUfePath = *ufePathIt;
         MFnDependencyNode dgNodeFn(dagPath.node());
 
+        // If the Maya node holds USD type information (e.g. a dummy transform
+        // node which is a stand-in for a non-transform USD prim type), use the
+        // USD type instead.
+        auto              usdTypeNamePlug = dgNodeFn.findPlug("USD_typeName", true);
+        bool              useUsdType = !usdTypeNamePlug.isNull();
         const std::string mayaTypeName(dgNodeFn.typeName().asChar());
```

Comment:
Really small opt: if we expect that usdTypeNamePlug will most often be valid, then moving the mayaTypeName code right inside the call to the other branch of the ?: op would avoid building that std::string most of the time.

Reply:
Makes perfect sense, done.